interaction_experiments/WfHub2LmInteractionExample.ipynb

```diff
@@ -0,0 +1,216 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Workflow Hub->LifeMonitor Interaction"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# required packages\n",
+    "import os\n",
+    "import requests\n",
+    "from pprint import pp"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# WhHub Settings\n",
+    "wfhub_username = \"workflowhub\"\n",
+    "wfhub_password = \"workflowhub\"\n",
+    "wfhub_url = \"__WFHUB_INSTANCE_BASE_URL__\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# LifeMonitor settings on WfHub\n",
+    "lm_base_url = \"__LIFE_MONITOR_INSTANCE_BASE_URL__\"\n",
+    "lm_login_url = f\"{lm_base_url}/oauth2/login/seek\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create a new session\n",
+    "s = requests.session()\n",
+    "s.headers.update({\n",
+    "    \"Content-type\": \"application/vnd.api+json\",\n",
+    "    \"Accept\": \"application/vnd.api+json\",\n",
+    "    \"Accept-Charset\": \"ISO-8859-1\"\n",
+    "})\n",
+    "# log the user on WfHub\n",
+    "s.auth=requests.auth.HTTPBasicAuth(wfhub_username, wfhub_password)\n",
+    "print(\"Cookies before the first call: %r\" % s.cookies)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# get userinfo from Workflow Hub\n",
+    "user_info = s.get(os.path.join(wfhub_url, 'people/current'))\n",
```

Comment:
Just a note that this and other WorkflowHub API calls, when testing things with self-signed certificates, need the `verify=False` param setting. E.g.:

```python
s.get(os.path.join(wfhub_url, 'people/current'), verify=False)
```

Reply:
Fixed by https://github.com/crs4/life_monitor/pull/32/commits/d74654f9259935b0e4acf4d006feaddfd10fd0d4